app/validation/validator.py

```diff
@@ -86,6 +88,27 @@ def _validate_json_against_schema(self, json_to_validate):
         except SchemaError as e:
             return '{}'.format(e)
 
+    def _validate_schema_contain_metadata(self, schema):
+
+        errors = []
+        default_metadata = ['user_id', 'period_id']
+        schema_metadata = schema['metadata']
+
+        # Find all words that precede with "metadata[' or meta': '"
+        parsed_metadata = re.findall(r"((?<=metadata\[\')\w+|(?<=meta\': \')\w+)", str(schema))
```

Comment:
Just a question for curiosity, is regex the best way to to find all metadata values

Reply:
In this case I think it makes sense. It matches the way that runner treats the whole schema as a template rather than just replacing values within specific values.